pages/examples/checklist-plots.py

```diff
@@ -0,0 +1,58 @@
+import plotly.express as px
+from dash import Dash, dcc, html
+from dash import Output, Input
+from plotly import data
+
+# Data
+df = data.iris()
+
+species = df["species"].unique().tolist()
+
+options = [
+    {'label': specie.capitalize(),
+     'value': specie} for specie in species]
+
+fig = px.scatter(
+        df,
+        x="sepal_length",
+        y="sepal_width", 
+        color="species"
+)
+
+color_discrete_map = {d.name: d.marker.color for d in fig.data}
+
+range_x = df["sepal_length"].agg({"min", "max"})
+range_y = df["sepal_width"].agg({"min", "max"})
+range_x = [range_x["min"]*.9, range_x["max"]*1.1]
+range_y = [range_y["min"]*.9, range_y["max"]*1.1]
+
+# App
+app = Dash(__name__)
+
+app.layout = html.Div([
+    dcc.Checklist(
+        options=options,
+        labelStyle={'display': 'inline-block'},
+        value=species,
+        id='checklist-plots-checklist'),
+     dcc.Graph(id='checklist-plots-scatter')
+])
+
+@app.callback(
+    Output('checklist-plots-scatter', 'figure'),
```

Comment:
As per the Readme ([name-of-the-file]-x-), I think the component id should be:
`Output('checklist-plots-x-scatter', 'figure')` And
`Input('checklist-plots-x-checklist', 'value')`

ID's should also be modified in the layout section, per component

Reply:
My bad here.